supplier/tests/test_supplier_page.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from unittest.mock import Mock
+from django.contrib.auth.models import User
+from supplier.views import suppliers_page
+
+
+@pytest.fixture
+def non_supplier_mock():
+    mock = Mock()
+    mock.supplier_account = User.objects.create_user(username='test',
```

Comment:
first name
last name
business name
needs to be saved as a supplier information

Reply:
Also user name set it to be higher than 6 because it will fail on the validation and not on the user does not exist
same for the password